src/player.js

```diff
@@ -794,6 +794,7 @@ export default class Player extends FakeEventTarget {
         sheet.insertRule(`#${this._playerId} .${SUBTITLES_CLASS_NAME} > div > div > div { ${style.toCSS()} }`, 0);
       }
       this._textStyle = style;
+      this.dispatchEvent(new FakeEvent(CustomEvents.TEXT_STYLE_CHANGED, {textStyle: this._textStyle.clone()}));
```

Comment:
Change the payload to:
` {textStyle: this.textStyle}`
it's already cloned the object,
so why access the private and clone it by yourself?

Reply:
irrelevant. i've removed the payload. taken from `textStyle` getter.